README.md

```diff
@@ -23,23 +23,26 @@
     python manage.py test
 - Run from main app folder in vagrant box where <i>manage.py</i> is present.
 - This will run both the unit tests as well as the PEP 8 tests.
+
+
+#### To run tests for a single app
+    python manage.py test *app_name*
+    
+- Can be run on: 'api', 'pep8' and 'blacklist_manager'
```

Comment:
If you want to run it on your own code you have to call it explicitly, since it is considered an "app", it's just not one we wrote.  I can move it to a section for third party apps?